docs/reference/commandline/context_import.md

```diff
@@ -19,4 +19,8 @@ keywords: "context, import"
 Usage:  docker context import [OPTIONS] CONTEXT FILE|-
 
 Import a context from a tar file
-```
\ No newline at end of file
+```
+
+## Description
+
+Imports a context previously exported with `docker context export`. To import from stdin, use `docker context import <context name> -`
```

Comment:
Perhaps explicitly mention the hyphen here (as it's easy to miss it at the end)

```suggestion
Imports a context previously exported with `docker context export`. To import
from `STDIN`, use a hyphen (`-`) as filename;

```bash
docker context import <context name> -
```

```